lib/seahorn/BvOpSem2.cc

```diff
@@ -885,6 +886,26 @@ class OpSemMemManager {
     return res;
   }
 
+  /// \brief Executes symbolic memcpy from physical memory with concrete length
+  Expr MemCpy(PtrTy dPtr, char* sPtr, unsigned len, uint32_t align) {
+    Expr res = m_ctx.read(m_ctx.getMemReadRegister());
+    unsigned sem_word_sz = wordSzInBytes();
+    for (int i = 0; i < len; i += sem_word_sz) {
```

Comment:
nit: since both `len` and `sem_word_sz` are `unsigned`, `i` should also be `unsigned`